metals/src/main/scala/scala/meta/internal/metals/debug/DebugServer.scala

```diff
@@ -133,6 +221,42 @@ object DebugServer {
     socket
   }
 
+  private def withRebuildRetry[A](
+      f: () => Try[A],
+      compilations: Compilations,
+      buildTargets: BuildTargets,
+      buildTargetClasses: BuildTargetClasses
+  )(implicit ec: ExecutionContext): Future[A] = {
+    Future.fromTry(f()).recoverWith {
+      case _: ClassNotFoundException | _: BuildTargetNotFoundException =>
```

Comment:
`BuildTargetNotFoundException` will not be needed here. User would need reimport the build to change them.

Reply:
good one, fixed